.travis.yml

```diff
@@ -58,6 +58,16 @@ jobs:
         on:
           tags: true
           repo: emberjs/ember-test-helpers
+    - if: tag IS present
+      script: yarn documentation
+      deploy:
+        provider: pages
+        skip_cleanup: true
+        local_dir: docs
+        github_token: $GITHUB_TOKEN # Set in travis-ci.org dashboard
```

Comment:
@rwjblue forgot to mention: this env var needs to be set by someone with sufficient privs (aka. not me 🤷‍♂️)